### ISSUE TITLE
Consecutive requires with no shared prefix flagged for multi-require

### ISSUE THREAD

**Issue by KronicDeth** [rrrene/credo#363]
With consective `require`s with no shared prefix, there is no syntactically valid way to combine the requires, but `Credo.Check.Consistency.MultiAliasImportRequireUse` is still saying they need to be combined.

### Code

> ```elixir
>   require Logger
>   require Query
> ```
> -- https://github.com/C-S-D/calcinator/blob/credo-363/lib/calcinator/resources/ecto/repo.ex#L52-L53

### Environment

* Credo version (`mix credo -v`): 0.7.3
* Erlang/Elixir version (`elixir -v`): 1.4.2
* Operating system: OSX

### What were you trying to do?

1. `git clone git@github.com:C-S-D/calcinator.git`
2. `cd calcinator`
3. `git checkout credo-363`
4. `mix deps.get`
5. `mix credo --strict`

### Expected outcome

Clean output

### Actual outcome

```

@lint attributes will be deprecated in the Credo v0.8 because they trigger
compiler warnings on Elixir v1.4.

Please consider reporting the cases where you needed @lint attributes
to help us devise a new solution: https://github.com/rrrene/credo/issues/new

Checking 27 source files ...

  Consistency                                                                   
┃ 
┃ [C] ↗ Most of the time you are using the multi-alias/require/import/use 
┃       syntax, but here you are using multiple single directives
┃       lib/calcinator/resources/ecto/repo.ex:53 (Calcinator.Resources.Ecto.Repo)

Please report incorrect results: https://github.com/rrrene/credo/issues

Analysis took 1.1 seconds (0.04s to load, 1.0s running checks)
163 mods/funs, found 1 consistency issue.
```

### Work-around

```patch
diff --git a/config/.credo.exs b/config/.credo.exs
index 57a0db0..58b1567 100644
--- a/config/.credo.exs
+++ b/config/.credo.exs
@@ -2,6 +2,7 @@
   configs: [
     %{
       checks: [
+        {Credo.Check.Consistency.MultiAliasImportRequireUse, false},
         {Credo.Check.Design.AliasUsage, excluded_lastnames: ~w(Controller), excluded_namespaces: ~w(Meta)},
         {Credo.Check.Readability.MaxLineLength, priority: :low, max_length: 120}
       ],
```

**Comment by rrrene** [comment]
I added a fix for this. Will be in the next release.

I have to admit that I like the idea of this check, but dislike the implementation. I should have put more thought into it 😢

**Comment by rrrene** [comment]
@KronicDeth I disabled the `MultiAliasImportRequireUse` by default. It is not up to speed (yet).

Thx for your bug report! :+1: Your steady contributions to this little project are greatly appreciated 😎
